compiler/modules/comm.lib/decimal.bal

```diff
@@ -0,0 +1,91 @@
+public function toDpd(decimal val) returns [int, int] {
```

Comment:
I don't understand byte ordering here.  I would expect this to be `int[2]` in either little or big endian order (and have a name that reflects this).  Or does DPD require a particular byte order?  `[top, bottom]` is strange because elsewhere in nBallerina we assume little-endian.

Reply:
fixed